tests/visual-snapshots.spec.ts

```diff
@@ -49,4 +49,6 @@ for (const { storyId, component } of stories) {
   }
 }
 
-handleUnusedScreenshots(usedScreenshotPaths);
+test.afterAll(() => {
+  handleUnusedScreenshots(usedScreenshotPaths);
+});
```

Comment:
I am pretty sure running this in `afterAll` isn't necessary. The `usedScreenshotPaths` array is populated before any tests are executed, so I don't think we have to wait until `afterAll` to handle unused screenshots.

Reply:
Good catch. Removed it.

Reply:
Actually, this breaks it. The `handleUnusedScreenshots` needs to know if it should update the snapshots or report a failure. It checks `process.argv.includes('--update-snapshots')` for this, but that doesn't work if this it is called in `afterAll`.

Reply:
I commented again without refreshing and did not see your comment. Sorry about that. More info, though.

Reply:
Thanks!